packages/manager/apps/pci-instances/public/translations/list/Messages_fr_FR.json

```diff
@@ -0,0 +1,17 @@
+{
+  "nameId": "Nom/ID",
+  "status": "Statut",
+  "publicIPs": "IP publique(s)",
+  "privateIPs": "IP privée(s)",
+  "cancel": "Annuler",
+  "image": "Image",
+  "flavor": "Modèle",
+  "instancesTitle": "Instances",
+  "createInstance": "Créer une instance",
+  "region": "Localisation",
+  "filter": "Filtrer",
+  "refresh": "Rafraîchir",
+  "inconsistencyMessage": "Toutes les données n'ont pas pu être chargées. Notre équipe technique est informée et travaille à résoudre le problème.",
+  "unknownErrorMessage1": "Nous n'avons pas pu récupérer la liste de vos instances. Veuillez cliquer sur le bouton <Link>'Rafraîchir'</Link> pour essayer à nouveau.",
+  "unknownErrorMessage2": "Nos équipes techniques ont été notifiées de ce problème et travaillent à sa résolution."
```

Comment:
**nitpick:** Use `snake-casing` for defining Translation Keys
```suggestion
  "instances_title": "Instances",
  "create_instance": "Créer une instance",
  "region": "Localisation",
  "filter": "Filtrer",
  "refresh": "Rafraîchir",
  "inconsistency_message": "Toutes les données n'ont pas pu être chargées. Notre équipe technique est informée et travaille à résoudre le problème.",
  "unknown_error_eessage1": "Nous n'avons pas pu récupérer la liste de vos instances. Veuillez cliquer sur le bouton <Link>'Rafraîchir'</Link> pour essayer à nouveau.",
  "unknown_error_message2": "Nos équipes techniques ont été notifiées de ce problème et travaillent à sa résolution."
```

Reply:
Thanks @anooparveti for your proposal, I modified translation keys as requested.